v2/pkg/engine/resolve/loader.go

```diff
@@ -846,16 +846,6 @@ func (l *Loader) renderRateLimitRejectedErrors(fetchItem *FetchItem, res *result
 }
 
 func (l *Loader) isFetchAuthorized(input []byte, info *FetchInfo, res *result) (authorized bool, err error) {
-	if info.OperationType == ast.OperationTypeQuery {
```

Comment:
is it changed?

Reply:
yes, a customer asked to run the check before